cfpb/cfgov-refresh
# Housing counselor: Update JS and cleanup

## Removals

- Removes jQuery from housing counselor page.

## Changes

- Removes unnecessary markup from housing counselor page.
- Removes unused styles from housing counselor page.
- Fixes JS for clicking map markers to scroll to result list items.
- Top aligns result list items.
- Moves map marker icons into find a housing counselor app assets.
- Adds more JS test coverage.
- Makes it so hud.js module doesn't auto-initialize the map (auto-initialization is an anti-pattern in my opinion).

## Testing

1. `gulp clean && gulp build`
2. Visit http://localhost:8000/find-a-housing-counselor/ and enter a zip code.
3. Click a map marker and observe it scrolls to the result item.
